data/com.github.tchx84.Flatseal.appdata.xml.in

```diff
@@ -32,10 +33,12 @@
   <update_contact>martin.abente.lahaye@gmail.com</update_contact>
   <!-- TRANSLATORS: Don't translate this text -->
   <developer_name>Martin Abente Lahaye</developer_name>
+  <developer id="com.github.tchx84">
+    <name translatable="no">Martin Abente Lahaye</name>
```

Comment:
I set this as translatable="no" based on the line above. Alternatively I could offer one of:

- Set the other as translatable="no" too and remove its comment
- Remove translatable="no" from this one, the comment will appear anyways in the .po file due to how gettext works (as long as it remains on top of the string above).

While there is no consensus on the matter, there is an argument in favor of allowing translations. It might be useful to provide a transliteration in certain languages.